### DISCUSSION TITLE
Is it possible to derive the Bernoulli equation from pipe flow CFD data with SINDY?

### DISCUSSION THREAD

**Discussion by atabakselahi** [dynamicslab/pysindy#278] [General]
I am trying to derive the Bernoulli equation from the CFD data that I have for Pipe flow. I am new to SINDY and am unsure how I should set up the model for this case.

Thank you so much for your help.

**Comment by azanparmar** [comment]
You can use the WeakPDE or PDE formulation
Check out the example of Burgers equation, it will help you to set it up.
There are examples of 2D and 3D PDE data too so that should suffice

**Reply by atabakselahi** [reply]
Thank you for your help. The burgers equation would be a great starting point.
